materiaali/python/viikko2.md

```diff
@@ -875,11 +875,10 @@ Tee testihakemistoon testitiedosto <i>kassapaate_test.py</i> ja sinne testiluokk
   - Jos maksu ei ole riittävä: kassassa oleva rahamäärä ei muutu, kaikki rahat palautetaan vaihtorahana ja myytyjen lounaiden määrässä ei muutosta
 - _seuraavissa testeissä tarvitaan myös Maksukorttia jonka oletetaan toimivan oikein_
 - Korttiosto toimii sekä edullisten että maukkaiden lounaiden osalta
-  - Jos kortilla on tarpeeksi rahaa, veloitetaan summa kortilta ja palautetaan _True_
```

Comment:
näitä ei poisteta. 

testin on pakko varmistaa että korttia veloitetaan oikein. jos näin ei tapahdu, voi kassapääte tehdä kortille ihan motä sattuu